csharp/src/Drivers/Apache/Hive2/HiveServer2Connection.cs

```diff
@@ -30,7 +30,7 @@ internal abstract class HiveServer2Connection : AdbcConnection
     {
         internal const long BatchSizeDefault = 50000;
         internal const int PollTimeMillisecondsDefault = 500;
-
+        private const int ConnectTimeoutMillisecondDefault = 30000;
```

Comment:
nit: `ConnectTimeoutMillisecondsDefault` for consistency with `PollTimeMillisecondsDefault`